apps/design-tokens-ui/src/app/app.component.scss

```diff
@@ -0,0 +1,84 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../../../../libs/barista-components/style/font-styles.scss';
+@import '../styles/variables.scss';
+
+:host {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+  font-family: 'BerninaSansWeb', 'OpenSans', sans-serif;
+}
+
+.nav {
+  top: 0;
+  width: 100%;
+  right: 0;
+  z-index: 1;
+}
+
+.main {
+  flex: 1 1 auto;
+}
+
+.sidenav-title {
+  display: inline-flex;
+  line-height: $top-bar-navigation-height;
+  font-size: 112%;
+}
+
+.sidenav-title-image {
+  height: 24px;
+  width: 24px;
+  fill: #ffffff;
+  line-height: 56px;
+  margin-right: 10px;
+}
+
+.top-bar-button {
+  width: 44px;
+  height: $top-bar-navigation-height;
+  fill: #ffffff;
+  background: #191919;
+  border: 0;
+  outline: none;
+  cursor: pointer;
+
+  &:hover {
+    background: #353535;
+  }
+}
+
+.breadcrumbs {
+  height: $top-bar-navigation-height;
+  display: inline-block;
+  width: 100%;
```

Comment:
It's inside a `top-bar-navigation`, so it looks of with the wrong height. I can't really change the top bar styling either since there's a button when you hide the left bar. I could take another look but I don't know if it makes sense since we've discussed that the styling will change later anyways.